pkg/convert/json_ngxtranslate_test.go

```diff
@@ -94,7 +94,8 @@ func Test_FromNgxTranslate(t *testing.T) {
 				return
 			}
 
-			assert.Equal(t, tt.expected, result)
+			assert.Equal(t, tt.expected.Language, result.Language)
+			assert.ElementsMatch(t, tt.expected.Messages, result.Messages)
```

Comment:
Fixed random test failing, due to `result.Messages` order could be different than in `tt.expected.Messages`, which cause `assert.Equal()` to fail

Reply:
In golang, the key order in the map is indeterministic.

Reply:
it is not a map, it is `[]model.Message`. 
```go
assert.Equal({msg1, msg2}, {msg2, msg1}) = false

assert.ElementsMatch({msg1, msg2}, {msg2, msg1}) = true
```

Reply:
yeah, but under the hood, it uses maps in the flow, doesn't it? :)

Reply:
did not catch that